python/ql/src/semmle/python/regex.qll

```diff
@@ -371,10 +371,13 @@ abstract class RegexString extends Expr {
       or
       // octal value \ooo
       end in [start + 2 .. start + 4] and
-      this.getText().substring(start + 1, end).toInt() >= 0 and
+      // this.isOctal([start + 1 .. end]) and
+      forall(int i | i in [start + 1 .. end - 1] | this.isOctal(i)) and
+      // this.getText().substring(start + 1, end).toInt() >= 0 and
       not (
         end < start + 4 and
-        exists(this.getText().substring(start + 1, end + 1).toInt())
+        this.isOctal(end) //and
+        // exists(this.getText().substring(start + 1, end + 1).toInt())
```

Comment:
Leftover comments? Did you forget to remove them?

Reply:
Indeed I did, thanks :-)